python/vowpalwabbit/sklearn_vw.py

```diff
@@ -672,6 +673,73 @@ class VWRegressor(VW, RegressorMixin):
 
     pass
 
+class ThresholdingMultiClassifierMixin(ClassifierMixin):
```

Comment:
I wouldn't bother creating a separate class, you can roll the predict method into VWMultiClassifier

Reply:
okay